pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -825,17 +826,19 @@ func PodPortsWithXsite(i *infinispanv1.Infinispan) []corev1.ContainerPort {
 }
 
 func PodLivenessProbe(i *infinispanv1.Infinispan) *corev1.Probe {
-	return &corev1.Probe{
-		Handler:             ispn.ClusterStatusHandler(i.GetEndpointURIScheme()),
-		FailureThreshold:    5,
-		InitialDelaySeconds: 10,
-		PeriodSeconds:       10,
-		SuccessThreshold:    1,
-		TimeoutSeconds:      80,
-	}
+	return probe(i, 5, 10, 10, 1, 80)
 }
 
 func PodReadinessProbe(i *infinispanv1.Infinispan) *corev1.Probe {
+	return probe(i, 5, 10, 10, 1, 80)
+}
+
+func PodStartupProbe(i *infinispanv1.Infinispan) *corev1.Probe {
+	// Maximum 10 minutes (60 * 10s) to finish startup
+	return probe(i, 60, 10, 10, 1, 80)
+}
+
+func probe(i *infinispanv1.Infinispan, failureThreshold, initialDelay, period, successThreshold, timeout int) *corev1.Probe {
```

Comment:
`failureThreshold, initialDelay, period, successThreshold, timeout int` parameters are not using

Reply:
Wow I must have been coding with my eyes closed!